src/components/block-tunes/block-tune-move-up.ts

```diff
@@ -90,8 +76,8 @@ export default class MoveUpTune implements BlockTune {
       return;
     }
 
-    const currentBlockElement = currentBlock.holder;
-    const previousBlockElement = previousBlock.holder;
+    const currentBlockElement = (currentBlock as BlockAPI).holder;
+    const previousBlockElement = (previousBlock as BlockAPI).holder;
```

Comment:
Looks like you could just do this:
```suggestion
    const currentBlockElement = currentBlock!.holder;
    const previousBlockElement = previousBlock!.holder;
```